src/Uno.UI.RuntimeTests/Tests/Windows_UI_Core/Given_CoreDispatcher.cs

```diff
@@ -0,0 +1,63 @@
+﻿
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Windows.Foundation;
+using Windows.System;
+using Windows.UI.Core;
+
+namespace Uno.UI.RuntimeTests.Tests.Windows_UI_Core
+{
+  #if __ANDROID__ || __IOS__
+  [TestClass]
```

Comment:
Thanks for the update! I'm curious though, the indendation is not correct, but it should have been fixed by the `.editorconfig` file. Do you know if its activated properly ?